### ISSUE TITLE
Rule idea: no-duplicate-at-functions: Disallow duplicate functions within a stylesheet

### ISSUE THREAD

**Issue by Grawl** [stylelint-scss/stylelint-scss#731]
There are two rules to disallow duplicates dollar variables and mixins:

> [no-duplicate-dollar-variables](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/no-duplicate-dollar-variables/README.md): Disallow duplicate dollar variables within a stylesheet.
> 
> [no-duplicate-mixins](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/no-duplicate-mixins/README.md): Disallow duplicate mixins within a stylesheet.

But there is no rule to disallow duplicate SCSS `@`-functions.

In my idea, following code patterns are considered warnings:

```scss
@function foo($arg) {
    @return $arg * 2;
}
@function foo($arg) {
    @return $arg * 4;
}
```

**Comment by kristerkari** [comment]
Thanks! I think that it indeed would make sense to have such rule.
